src/languages/python.js

```diff
@@ -39,43 +39,43 @@ export default function(hljs) {
     contains: [hljs.BACKSLASH_ESCAPE],
     variants: [
       {
-        begin: /(u|b)?r?'''/, end: /'''/,
+        begin: /([uU]|[bB]|[rR]|[bB][rR]|[rR][bB])?'''/, end: /'''/,
         contains: [hljs.BACKSLASH_ESCAPE, PROMPT],
         relevance: 10
       },
       {
-        begin: /(u|b)?r?"""/, end: /"""/,
+        begin: /([uU]|[bB]|[rR]|[bB][rR]|[rR][bB])?"""/, end: /"""/,
         contains: [hljs.BACKSLASH_ESCAPE, PROMPT],
         relevance: 10
       },
       {
-        begin: /(fr|rf|f)'''/, end: /'''/,
+        begin: /([fF][rR]|[rR][fF]|[fF])'''/, end: /'''/,
         contains: [hljs.BACKSLASH_ESCAPE, PROMPT, LITERAL_BRACKET, SUBST]
       },
       {
-        begin: /(fr|rf|f)"""/, end: /"""/,
+        begin: /([fF][rR]|[rR][fF]|[fF])"""/, end: /"""/,
         contains: [hljs.BACKSLASH_ESCAPE, PROMPT, LITERAL_BRACKET, SUBST]
       },
       {
-        begin: /(u|r|ur)'/, end: /'/,
+        begin: /([uU]|[rR])'/, end: /'/,
         relevance: 10
       },
       {
-        begin: /(u|r|ur)"/, end: /"/,
+        begin: /([uU]|[rR])"/, end: /"/,
         relevance: 10
       },
       {
-        begin: /(b|br)'/, end: /'/
+        begin: /([bB][rR]?|[rR][bB])'/, end: /'/
```

Comment:
Isn't this inconsistent with how you write out the combos below?  Why in once case do we use an optional and in the other case we just list all 3 combos explicitly?  I'm not seeing the distinction.

Reply:
Yeah, that’s an oversight.